### ISSUE TITLE
[Bug]: Permalink issues with de_DE 1.0.1

### ISSUE THREAD

**Issue by sterbekuenstler** [bueltge/de-DE#19]
### Description of the bug

When testing version 1.0.1 in various systems, I noticed that there seems to be a bug with the permalinks. I created a post with umlauts and got problems with umlauts. When I deactivated the plugin, the error was no longer present.

With the test file "The ugly duckling costs 1 €.jpg" the result looks like this: File name: "das-haessliche-entlein-kostet-1-e-scaled.jpg".

A post/page with the name "Das hässliche Entlein kostet 1 €.jpg" becomes "das-ha¤aliche-e...ein-kostet-1-eur".

Error can be reproduced 1:1 in various similar system setups. 

### Reproduction instructions

1. Update de_DE to 1.0.1
2. Generate File, Post or Page with Title "Das häßliche Entlein kostet 1 €.jpg"

### Expected behavior

I did not expect any intervention in the permalink structure of posts and pages. File names seem to work at first glance.

### Environment info

PHP 8.2 + 8.3
Avada 7.11.9
WordPress 6.5.4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bueltge** [comment]
@sterbekuenstler Would you tr it again with the new version 1.0.2? Thanks.

**Comment by sterbekuenstler** [comment]
@bueltge I was able to test the new version yesterday on i could test macOS Sonoma 14.6.1 (23G93) with Safari 17.6, Google Chrome version 128.0.6613.120, it worked.

On Windows 11 Pro 23H2 on Google Chrome version 129.0.6668.59 and Edge version 129.0.2792.52 it also worked.

In a test post, the title "Das häßliche Entlein kostet 1 €" to the permalink "das-haessliche-entlein-kostet-1-eur"

and an image was changed from "Das häßliche Entlein kostet 1 €.jpg" to "das-haessliche-entlein-kostet-1-e.jpg".

It looks functional again to me.
